### DISCUSSION TITLE
Should nextjs integration always use expo?

### DISCUSSION THREAD

**Discussion by Wuyialex** [nandorojo/dripsy#141] [Q&A]
Looking at the docs and examples, it seems that you should always use expo/next-adapter. Can't I use dripsy with bare react native project(not use expo) and nextjs?

**Comment by nandorojo** [comment]
Yeah you can. In fact, I think it might work with SWC directly without a Babel config on the next side if you don’t need to use Reanimated. Did you look into the monorepo starter though? Expo’s Babel config has many optimizations for RN packages to be used on web.
